package.json

```diff
@@ -32,8 +32,8 @@
   ],
   "dependencies": {
     "@webex/component-adapter-interfaces": "^1.30.5",
-    "@webex/sdk-component-adapter": "1.112.6",
     "@webex/components": "^1.274.2",
+    "@webex/sdk-component-adapter": "1.112.6",
```

Comment:
Any reason for this change?

Reply:
No reason, this change came on it's own somehow